Backend/Remora.Discord.API.Abstractions/API/Objects/Interactions/Components/ComponentType.cs

```diff
@@ -43,10 +43,30 @@ public enum ComponentType
     /// <summary>
     /// A menu of selectable options.
```

Comment:
"options" should probably be "values" or "strings" here.

Reply:
Will update to 'strings'.